python/pipeline/meso.py

```diff
@@ -266,56 +268,73 @@ def _make_tuples(self, key):
             correction_channel = (CorrectionChannel() & key & {'field': field_id + 1})
             channel = correction_channel.fetch1('channel') - 1
 
-            # Create results tuple
-            tuple_ = key.copy()
-            tuple_['field'] = field_id + 1
-
-            # Load scan (we discard some rows/cols to avoid edge artifacts)
-            skip_rows = int(round(px_height * 0.10))
+            # Load some frames from middle of scan to compute template
+            skip_rows = int(round(px_height * 0.10)) # we discard some rows/cols to avoid edge artifacts
             skip_cols = int(round(px_width * 0.10))
-            scan_ = scan[field_id, skip_rows: -skip_rows, skip_cols: -skip_cols, channel, :]  # height x width x frames
-            scan_ = scan_.astype(np.float32, copy=False)
+            middle_frame = int(np.floor(scan.num_frames / 2))
+            mini_scan = scan[field_id, skip_rows:-skip_rows, skip_cols: -skip_cols,
+                             channel, max(middle_frame - 1000, 0): middle_frame + 1000]
+            mini_scan = mini_scan.astype(np.float32, copy=False)
 
-            # Correct raster effects (needed for subpixel changes in y)
+            # Correct mini scan
             correct_raster = (RasterCorrection() & key & {'field': field_id + 1}).get_correct_raster()
-            scan_ = correct_raster(scan_)
-            scan_ -= scan_.min()  # make nonnegative for fft
+            mini_scan = correct_raster(mini_scan)
 
             # Create template
-            middle_frame = int(np.floor(scan.num_frames / 2))
-            mini_scan = scan_[:, :, max(middle_frame - 1000, 0): middle_frame + 1000]
-            mini_scan = 2 * np.sqrt(mini_scan + 3 / 8)  # *
```

Comment:
comment Anscombe transform

Reply:
?, just put anscombe transform, rather than the * and the explanation below?

Reply:
Oh, i see.  I just did not understand the `# *` notation